mkosi/distributions/none.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import urllib.parse
+from collections.abc import Sequence
+
+from mkosi.architecture import Architecture
+from mkosi.distributions import Distribution, DistributionInstaller, PackageType
+from mkosi.log import die
+from mkosi.state import MkosiState
+
+
+class NoneInstaller(DistributionInstaller):
+    @classmethod
+    def filesystem(cls) -> str:
+        return "ext4"
+
+    @classmethod
+    def package_type(cls) -> PackageType:
+        return PackageType.rpm
+
+    @classmethod
+    def default_release(cls) -> str:
+        return "0"
+
+    @classmethod
+    def default_tools_tree_distribution(cls) -> Distribution:
+        return Distribution.none
+
+    @classmethod
+    def tools_tree_packages(cls) -> list[str]:
+        return []
+
+    @classmethod
+    def setup(cls, state: MkosiState) -> None:
+        pass
+
+    @classmethod
+    def install(cls, state: MkosiState) -> None:
+        pass
+
+    @classmethod
+    def install_packages(cls, state: MkosiState, packages: Sequence[str], apivfs: bool = True) -> None:
+        pass
+
+    @classmethod
+    def remove_packages(cls, state: MkosiState, packages: Sequence[str]) -> None:
+        pass
+
+    @staticmethod
+    def architecture(arch: Architecture) -> str:
+        a = {
+            Architecture.arm64     : "aarch64",
+            Architecture.ia64      : "ia64",
+            Architecture.mips64_le : "mips64el",
+            Architecture.mips_le   : "mipsel",
+            Architecture.parisc    : "parisc64",
+            Architecture.ppc64_le  : "ppc64le",
+            Architecture.riscv64   : "riscv64",
+            Architecture.s390x     : "s390x",
+            Architecture.x86_64    : "x86_64",
+        }.get(arch)
+
+        if not a:
+            die(f"Architecture {a} is not supported")
+
+        return a
```

Comment:
I see that it's taken from Fedora, by without making further any reason not do just do
```suggestion
        return str(arch)
```